iroha/src/sumeragi.rs

```diff
@@ -149,11 +150,13 @@ impl Sumeragi {
             self.validating_peers(),
         )
         .await;
-        let _result = Peer::send(
-            peer::Message::SumeragiMessage(Message::Created(block.clone())),
-            self.proxy_tail().clone(),
-        )
-        .await;
+        let _result = Network::send_request_to(
+            self.proxy_tail().address.as_ref(),
+            Request::new(
+                "/blocks".to_string(),
```

Comment:
Here and in other requests you send to `/blocks` but `torii` listens on `/block`. I think we need to make the endpoint urls in torii public and not use raw values in code.